CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contribute
+
```

Comment:
Need to add a bit of introduction, something like:

There are several ways you can contribute to GlotPress and all are appreciated:

1. Submitting patches: whether fixing a bug or adding new functionality
2. Testing patches / PR's
3. Running and writing tests